drivers/media/i2c/ov5693.c

```diff
@@ -1617,24 +1305,18 @@ static int ov5693_init_controls(struct ov5693_device *ov5693)
 	struct i2c_client *client = v4l2_get_subdevdata(&ov5693->sd);
 	const struct v4l2_ctrl_ops *ops = &ov5693_ctrl_ops;
 	struct v4l2_fwnode_device_properties props;
-	unsigned int i;
 	int ret;
 	int hblank;
 	int vblank_max, vblank_min, vblank_def;
 	int exposure_max;
 
 	ret = v4l2_ctrl_handler_init(&ov5693->ctrl_handler,
-				     ARRAY_SIZE(ov5693_controls));
+				     8);
```

Comment:
You're right, that was a bit lazy, my bad. In a one-of-a-kind experience for me this parameter is specifically an _estimate_, so I guess I could do something like OV5693_APPROX_NUM_CTRLS (it works if the answer's wrong, just takes a bit longer).

Reply:
See docs:
```
 * @nr_of_controls_hint: A hint of how many controls this handler is
 *		expected to refer to. This is the total number, so including
 *		any inherited controls. It doesn't have to be precise, but if
 *		it is way off, then you either waste memory (too many buckets
 *		are allocated) or the control lookup becomes slower (not enough
 *		buckets are allocated, so there are more slow list lookups).
 *		It will always work, though.
```
Other drivers use a plain number too so I would leave it as is.

Reply:
Okay.

Reply:
Fixed the formatting, leaving the magic number :)